src/backend/app/auth/auth_routes.py

```diff
@@ -81,28 +91,21 @@ async def callback(request: Request, osm_auth=Depends(init_osm_auth)):
     # Get access token
     access_token = osm_auth.callback(callback_url).get("access_token")
     log.debug(f"Access token returned of length {len(access_token)}")
-    response = Response(status_code=HTTPStatus.OK)
 
-    # Set cookie
-    cookie_name = settings.FMTM_DOMAIN.replace(".", "_")
-    log.debug(
-        f"Setting cookie in response named '{cookie_name}' with params: "
-        f"max_age=31536000 | expires=31536000 | path='/' | "
-        f"domain={settings.FMTM_DOMAIN} | httponly=True | samesite='lax' | "
-        f"secure={False if settings.DEBUG else True}"
-    )
-    response.set_cookie(
-        key=cookie_name,
-        value=access_token,
-        max_age=31536000,  # OSM currently has no expiry
-        expires=31536000,  # OSM currently has no expiry
-        path="/",
-        domain=settings.FMTM_DOMAIN,
-        secure=False if settings.DEBUG else True,
-        httponly=True,
-        samesite="lax",
-    )
-    return response
+    osm_user = osm_auth.deserialize_access_token(access_token)
+    user_data = {
+        "id": osm_user["id"],
+        "sub": f"fmtm|{osm_user['id']}",
```

Comment:
Can the sub be changed to `osm|xxx` here?

The temp auth would be something like `fmtm|temp`

Reply:
`osm|username` or `osm|id` ?

Reply:
The sub should always have the id, so good point what I wrote for the temp auth is inconsistent. 

osm|123 or fmtm|123

We have a username claim for the username

Reply:
So if we aren't using svcfmtm, then we need a temporary user profile and its id to use as sub.